sycl/plugins/level_zero/pi_level_zero.cpp

```diff
@@ -87,6 +87,12 @@ pi_result piDeviceRelease(pi_device Device) {
 pi_result piDeviceGetInfo(pi_device Device, pi_device_info ParamName,
                           size_t ParamValueSize, void *ParamValue,
                           size_t *ParamValueSizeRet) {
+  // TODO: This is a work-around for missing UR info query. Replace with propper
```

Comment:
I think this is good, hardcoding it. But this should go in urDeviceGetInfo in the L0 adapter. Here, we only redirect.

https://github.com/intel/llvm/blob/474461cb2e1de6f4e63a5d7d6edb6e93cb37b486/sycl/plugins/unified_runtime/ur/adapters/level_zero/device.cpp#L89

Reply:
Agreed! I realized a little late that we missed the support enumerator when porting it to UR. It has since been amended and it should now be handled by the adapters, so I have removed this. Good eye!